website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -1154,6 +1154,25 @@ eventHandlers:
 
 Inside the handler function, the receipt can be accessed in the `Event.receipt` field. When the `receipt` key is set to `false` or omitted in the manifest, a `null` value will be returned instead.
 
+## Declared eth_call
+
+Reducing `eth_call` definitely improves indexing speed. For `eth_calls` that cannot be eliminated, they can be declared in the subgraph manifest to be processed in parallel with the block processing. When `graph-node` processes a block, it performs all declared `eth_calls` in parallel before handlers are run where as `eth_calls` that are not declared are executed sequentially when handlers run. This helps reducing total time spent on `eth_calls` and improves indexing speed.
+
+> **Note:** [SpecVersion](#specversion-releases) >= `1.2.0` is required. Currently, `eth_calls` can only be declared for event handlers.
+
+### Example Configuration in Subgraph Manifest
+
+```yaml
+event: TransferWithPool(address indexed, address indexed, uint256, bytes32 indexed)
+handler: handleTransferWithPool
+calls:
+  ERC20.poolInfo: ERC20[event.address].getPoolInfo(event.params.to)
+```
+
+In the example above, the `ERC20.poolInfo` is declared `eth_call`. The text before colon(`ERC20.poolInfo`) is the label for this `eth_call` which is used when logging errors. The text after colon(`ERC20[event.address].getPoolInfo(event.params.to)`) is the actual `eth_call` that will be executed which is in the form of `Contract[address].function(arguments)`. The `address` and `arguments` can be replaced with actual values or variables which will be available when executing the handler.
```

Comment:
> The `address` and `arguments` can be replaced with actual values or variables which will be available when executing the handler.

I think per [this comment](https://github.com/graphprotocol/graph-node/issues/5445#issuecomment-2145722979) you can't pass in hard-coded values currently. The above kind of suggests that you can, so maybe remove "actual values or"?